Kernel/Syscall.cpp

```diff
@@ -183,9 +183,8 @@ static u32 handle(RegisterDump& regs, u32 function, u32 arg1, u32 arg2, u32 arg3
     case Syscall::SC_sigreturn:
         if (auto* tracer = current->process().tracer())
```

Comment:
These two lines should be removed. They were only necessary before because we did not return from `sys$sigreturn()`.
Now that we return, the trap entry function will do this work instead, like with most other syscalls.